Source/multi.cpp

```diff
@@ -237,7 +237,7 @@ void PlayerLeftMsg(int pnum, bool left)
 		return;
 	}
 
-	RemovePlrFromMap(pnum);
+	FixPlrWalkTags(player);

```

Comment:
This does make the assumption that there was never a need to scan the full map for a bad player position. But maybe we should just track down such cases and fix them if they exists.